pkg/metrics/metrics.go

```diff
@@ -60,12 +60,25 @@ var (
 		},
 		[]string{"broker"},
 	)
+
+	// OSBRequestCount exposes the number of HTTP requests made to Open Service
+	// Brokers.  The metric is broken out by broker name and response status
+	// (100/200/300/400/500 or 'client-error')
+	OSBRequestCount = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: catalogNamespace,
+			Name:      "osb_request_count",
+			Help:      "Cumulative number of HTTP requests from the OSB Client to the specified Service Broker grouped by name and response status.",
+		},
+		[]string{"broker", "status"},
```

Comment:
What do you think about breaking this out one level deeper into request type as well? Would that be too much burden on the counters?

Reply:
@staebler can you elaborate on request type?  Are you talking about put/post/get/delete/patch?

Reply:
Catalog, Provision, Deprovision, etc.

Reply:
Good call Matt, done.  I played around with the result in Prometheus, I think its a good addition, but time and use will determine if its too unwieldy.  I think it's worth a try, you get some much better insights.